imageio/plugins/_freeimage.py

```diff
@@ -953,28 +953,20 @@ def set_image_data(self, array):
             n_channels = shape[0]
         
         def n(arr):  # normalise to freeimage's in-memory format
-            return arr.T[:, ::-1]
+            return arr[::-1].T
```

Comment:
I don't see how this would fix it. The net result should be the same. I verfied that it is for a png image. Maybe for exr it does make a difference? Is it not 2D?